src/nestLists.ts

```diff
@@ -60,6 +60,9 @@ export function nestLists<T extends TypedObject = PortableTextBlock | TypedObjec
 
   for (let i = 0; i < blocks.length; i++) {
     const block = blocks[i]
+    if (!block) {
+      continue
```

Comment:
This is meant to guard the block from being `undefined`. Perhaps throwing is better than `continue`.